ansible/roles/kubernetes/files/make-ca-cert.sh

```diff
@@ -134,6 +134,7 @@ fi
 mkdir -p "$cert_dir"
 
 cp -p pki/ca.crt "${cert_dir}/ca.crt"
+cp -p pki/private/ca.key "${cert_dir}/ca.key"
```

Comment:
Why not add a condition here and use an environment variable for this in the Ansible task?